jenkins/Makefile

```diff
@@ -45,7 +45,7 @@ RPMBUILD_BASEIMAGE_REPO = $(subst ${VER},${DIST_VERSION},${RPMBUILD_BASEIMAGE_${
 #
 # TODO:  The Jinja2 CLI is not currently available as an RPM on RHEL, so we
 #        install it via pip for all platforms, to keep things simple; we should
-#        reevaluate this (and the other) dependencies in the future.
+#        reevaluate this dependency (and the other dependencies) in the future.
```

Comment:
"this dependency and the other dependencies" sounds a bit ... redundant? Perhaps just "re-evaluate our dependencies"??